### PR TITLE
[EateryBlue Refactor] [1/4] Restructured eatery-related files into MVC design pattern

### PR DESCRIPTION
**Pull request by Archit404Error**
## Overview
This PR creates a new app called eatery which contains all models, views, and controllers that deal with eateries. I pulled these out of the central api app. Additionally, I pulled out all datatypes from api/ that pertained to eateries. This is one PR out of several that need to be made in order to further decouple the codebase and fully transition to MVC.

## Changes Made
The new eatery app has the following structure:
Models:
- EateryStore

Views:
- UpdateView

Controllers:
- UpdateEateryController

Datatypes:
- Eatery
- EateryAlert

## Test Coverage

Tested by running bash reset_db.sh to apply migrations. Then, because the newly created eatery_eaterystore table was empty and I had some other data linked to the old eatery table via foreign keys, I opened the sqlite file in my database browser and ran `INSERT INTO eatery_eaterystore SELECT * FROM api_eaterystore` before running `bash reset_db.sh` a second time to fully apply all remaining migrations (errored out the first time due to integrity error).

Then I went to /api and ensured everything worked as expected including the repeating schedules and schedule exceptions. I also ran a POST request to /update to make sure it also still worked correctly.

## Next Steps (delete if not applicable)
Factor out other views/models/controllers into distinct apps. Specifically:
- Reports
- Schedules
- Menus

### CODE REVIEW

File: src/api/admin.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 
 import api.models as models
 
 # Register your models here.
 
-admin.site.register(models.EateryStore)
+admin.site.register(eatery_models.EateryStore)
 admin.site.register(models.MenuStore)
 admin.site.register(models.ItemStore)
 admin.site.register(models.SubItemStore)
```
Comment (alanna-zhou) [inline src/api/admin.py]: you should be registering admin under `src/eatery/admin.py`

**Review by alanna-zhou — CHANGES_REQUESTED** [review]
this PR looks like it has formatting stuff as well. could you make a separate pr just for formatting? it's difficult to review the stuff that is important

**Review by chalo2000 — APPROVED** [review]
Since I approved the unformatted version, I'll be approving this one as well.

**Review by alanna-zhou — APPROVED** [review]
nice!!

**Review by markim21 — APPROVED** [review]
Same comments as before! Looks good! :)

**Pull request merged**